datadog_sync/model/logs_pipelines.py

```diff
@@ -48,18 +50,42 @@ async def pre_apply_hook(self) -> None:
         self.destination_integration_pipelines = await self.get_destination_integration_pipelines()
 
     async def create_resource(self, _id: str, resource: Dict) -> Tuple[str, Dict]:
+        destination_client = self.config.destination_client
+
         if resource["is_read_only"]:
             if resource["name"] not in self.destination_integration_pipelines:
-                raise SkipResource(
-                    _id,
-                    self.resource_type,
-                    "integration pipelines cannot be created only updated. "
-                    + f"Skipping sync. Enable integration pipeline {resource['name']}",
-                )
+                # Extract the source from the query
+                source = resource.get("filter", {}).get("query", "").split("source:")[-1]
+                if not source:
+                    raise Exception(f"Source not found in the query for integration pipeline '{resource['name']}'")
+                payload = {
+                    "ddsource": source,
+                    "ddtags": ",".join(DEFAULT_TAGS),
+                    "message": f"[datadog-sync-cli] Triggering creation of '{resource['name']}' integration pipeline",
+                }
+
+                # Submit a log to the logs intake API to trigger the creation of the integration pipeline
+                await destination_client.post(self.logs_intake_path, payload, subdomain=self.logs_intake_subdomain)
+                created = False
+                for _ in range(3):
+                    updated_pipelines = await self.get_destination_integration_pipelines()
+                    if resource["name"] in updated_pipelines:
+                        self.destination_integration_pipelines = updated_pipelines
+                        created = True
+                        break
+                    else:
+                        await sleep(5)
+
+                if not created:
+                    raise Exception(f"Max retry reached. Integration pipeline '{resource['name']}' pending creation")
+
             self.resource_config.destination_resources[_id] = self.destination_integration_pipelines[resource["name"]]
-            return await self.update_resource(_id, resource)
+            diff = check_diff(self.resource_config, self.destination_integration_pipelines[resource["name"]], resource)
+            if diff:
+                return await self.update_resource(_id, resource)
```

Comment:
I wonder why we try to update resource here? this is the creation endpoint, right?

BTW, there is no need to update integration pipeline at all. There are read only.

Reply:
The update is to sync `enables/disables` status of the integration pipelines

Reply:
👍 Ah, ok, I get it now. Thanks for the explanation! So, probably it's better to add a comment here in the code about the intention of the update.